src/model/model.ts

```diff
@@ -1228,7 +1229,16 @@ function getDefaultHistoryQueryOpts(): HistoryQueryOpts {
 class InputModel {
     historyShow: OV<boolean> = mobx.observable.box(false);
     infoShow: OV<boolean> = mobx.observable.box(false);
+    aIChatShow: OV<boolean> = mobx.observable.box(false);
     cmdInputHeight: OV<number> = mobx.observable.box(0);
+    aiChatTextAreaRef: React.RefObject<HTMLTextAreaElement>;
+    codeSelectBlockRefArray: Array<React.RefObject<HTMLElement>>;
+    codeSelectMarkdownID: number;
```

Comment:
Currently markdown id makes it so that only the most recent query will be part of the selection. 
I did this initially because I figured the past queries wouldn't be relevant and I thought it had potential for a big memory load, but I think we might want to rethink this and just have every code block generated as part of this selection

Reply:
addressed, no more markdown id